src/axom/sidre/docs/sphinx/group.rst

```diff
@@ -50,7 +51,9 @@ the data is shared by the original and new View.
 .. note:: View copy operations perform **shallow** copies of the View data.
 
 Some methods for for creating, destroying, querying, and retrieving Groups and 
-Views take a string with *path syntax*, while others take the name of an
+Views take a string with *path syntax*, where parent and child Group names
+are joined with the path separator character (usually '/').
```

Comment:
Side question: Do we have any tests that modify the "path separator character" to something other than "/", e.g. ":"? (I suspect not).

Reply:
In this case I don't think the path separator is configurable?

That would only make sense for file path utilities, not sidre paths?

Reply:
> are joined with the path separator character *(usually '/')*. 

It looks  like we defined a static variable, but it cannot be changed.

https://github.com/LLNL/axom/blob/18b70302ab57d4f4bbaf6caa939ca6216f03c463/src/axom/sidre/core/Group.cpp#L30

In that case, should we rephrase the parenthetical at the end?

Reply:
changed to "the path separator character '/'."